bundles/framework/layerlist/view/LayerViewTabs/LayerViewTabsHandler.js

```diff
@@ -98,6 +98,19 @@ class UIStateHandler extends StateHandler {
                     this.getLayerListHandler().getFilterHandler().useStashedState();
                 }
             },
+            'MapLayerEvent': event => {
+                if (!['update', 'sticky', 'tool'].includes(event.getOperation())) {
```

Comment:
Shouldn't the layers be updated after these as well? Or is it handled somewhere else?

Reply:
For example on update the layer name might have changed or with tool another tool have been added.

Reply:
Yes. The condition is actually negated, so update, sticky and tool will update the state.